packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -119,14 +130,21 @@ export default class InsertTableView extends View {
 		this.on( 'change:rows', () => {
 			this._highlightGridBoxes();
 		} );
+
+		addKeyboardHandlingForGrid( this, this.items, 10, true );
+	}
+
+	render() {
+		super.render();
+
+		this.keystrokes.listenTo( this.element );
 	}
 
 	/**
 	 * @inheritDoc
 	 */
 	focus() {
-		// The dropdown panel expects DropdownPanelFocusable interface on views passed to dropdown panel. See #30.
-		// The method should be implemented while working on keyboard support for this view. See #22.
+		this.items.get( 0 ).focus();
```

Comment:
A CSS-related comment but I'm leaving it around here: I noticed the focused grid element has a weird (bold) outline. I think we should get rid of it, the mouse and keyboard selections should look the same IMO.

<img width="210" alt="image" src="https://user-images.githubusercontent.com/1099479/182323739-896f53a8-5879-458d-a774-5f3d5cccacdc.png">

Reply:
(Nothing critical, this could be a follow-up) 

Also, the `'boxover'` logic, should focus the grid item. Use case:

1.  Open the dropdown
2.  Select some geometry using a mouse
3.  Start playing with arrow keys

**Expected**: Arrow keys should take over and modify existing geometry.

**Actual**: They always start from 1x1

![](https://user-images.githubusercontent.com/1099479/182324619-6f8f3a58-3dd5-49ad-af31-ecdc8c8b60b9.gif)

Reply:
Good point :+1:

Reply:
Outline will be handled in #12256.